configs/fine-tune-llm_v2/gce.yaml

```diff
@@ -0,0 +1,15 @@
+head_node_type:
+  name: head
+  instance_type: n2-standard-8
+auto_select_worker_config: true
+worker_node_types: []
+
+gcp_advanced_configurations_json:
+  instance_properties:
+    labels:
+      as-feature-multi-zone: 'true'
```

Comment:
Shouldn't this be multi-zone: **false** to force single zone launches for FT? Otherwise, we can use basic-serverless-config instead of defining a new one.

Reply:
Not sure what `basic-serverless-config ` refers to.

`multi-zone` was true in case instance types are not available on a single zone on the platform side. Open to not having it on OA if it is atypical. I see this config under the endpoints_v2 template too

Reply:
We should set multi-zone to False for fine-tuning, otherwise allreduce traffic will cross zones, which can make the job 20x more expensive. Customers have run into this before.

Reply:
+1 to Eric's comment

Reply:
We're going to optimize autoscaling to allow launching workers in a different region than head for capacity issues

Reply:
ack. Updated the aws to use serverless config but keeping the gce one for now. Will have a separate thread with Shomil